.circleci/config.yml

```
@@ -110,16 +122,39 @@ jobs:
           name: Shpec unit tests on heroku-<<parameters.stack-version>>
           command: shpec << parameters.buildpack-dir >>/shpec/*_shpec.sh
 
+  cargo-test:
+    machine:
+      image: ubuntu-2004:202010-01
+    steps:
+      - checkout
+      - heroku-buildpacks/install-build-dependencies
+      - heroku-buildpacks/install-rust-toolchain
+      - run:
+          command: cargo test
```

https://doc.rust-lang.org/cargo/commands/cargo-check.html#manifest-options

```suggestion
          command: cargo test --locked
```